src/services/logout-service.ts

```diff
@@ -1,65 +1,65 @@
 import * as crypto from 'crypto'
 import chalk from 'chalk'
-import axios, {AxiosRequestConfig} from 'axios'
-import {vars} from '../vars'
-import {CliUx} from '@oclif/core'
-import {LogService} from '../services/log-service'
-import {AbortService} from '../services/abort-service'
+import axios, { AxiosRequestConfig } from 'axios'
```

Comment:
all these extra spaces, can you remove those back to how things were - to keep the same style. i know this is getting nitpicky, but let's do that until ts-standard gets added to the repo.